UI/Services/CookieBasedTuitionPartnerShortlistStorageService.cs

```diff
@@ -4,41 +4,41 @@
 namespace UI.Services;
 
 /// <summary>
-/// The CookieBasedTuitionPartnerShortlistStorageService stores ShortlistedTuitionPartner details in a cookie
+/// The CookieBasedTuitionPartnerCompareListStorageService stores CompareListedTuitionPartner details in a cookie
```

Comment:
Can we change the file name as well? i.e: `CookieBasedTuitionPartnerCompareListStorageService.cs`

Reply:
Done